src/python.cpp

```diff
@@ -928,6 +928,28 @@ bool py_is_callable(PyObjectRef x) {
     return py_is_callable(x.get());
 }
 
+// caches np.nditer function so we don't need to obtain it everytime we want to
+// cast numpy string arrays into R objects.
+PyObject* np_nditer;
```

Comment:
Style nit, can this please be static defined inside the function rather than a global?